tasks/windows_client.yml

```diff
@@ -0,0 +1,102 @@
+---
+# tasks/windows_client.yml: Deploy various client-side configurations for Sensu-client on Windows
+# This contains all tasks, but these should be moved off into their respective places.
+
+    #- include_vars: "{{ ansible_distribution }}.yml"
+  - include_vars: "{{ ansible_os_family }}.yml"
+
+  - name: Download the Sensu installer msi into a local dir
+    win_get_url:
+      url: "{{ sensu_msi_download_path }}"
+      dest: "{{ sensu_win_temp_path }}\\sensu-installer.msi"
+
+    # If a version already exists on the target system, it should be made absent first.
+  - name: Install sensu-client from local dir
+    win_package:
+      path: "{{ sensu_win_temp_path }}\\sensu-installer.msi"
+      product_id: "{{ sensu_msi_product_id }}"
+      arguments: /qn
+      force_extension: msi
+      validate_certs: no
```

Comment:
This bit worries me a tad, currently, you're using a local file which makes me think this might not even be required, do you happen to have the error when this was set to `yes` ?

Reply:
I don't really know windows but I know on our old on premise product we had to certify our installers to avoid the warnings about installing untrusted software or something like that. I am guessing this is needed when the smi does not have a signed installer?